Dockerfile-cli.template

```diff
@@ -65,6 +80,20 @@ RUN set -eux; \
 {{ if $key == "compose" then ( -}}
 	ln -sv "$plugin" /usr/local/bin/; \
 	docker-{{ $key }} --version; \
+	\
+	echo '{{
+		{
+			name: ("docker-" + $key),
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: "3.18"
+			},
+			licenses: [
+				"Apache-2.0"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/docker-{{ $key }}.spdx.json ; \
```

Comment:
This is only getting added for `docker-compose`, but the other binary of the loop is getting missed (`buildx`). Probably just needs to be moved above the `if $key == "compose"` line?